bokeh/command/__init__.py

```diff
@@ -50,6 +50,9 @@ def build_applications(self, args):
             else:
                 handler = ScriptHandler(filename=file)
 
+            if handler.failed:
+                die("Error running application handler %r: %s %s " % (handler, handler.error, handler.error_detail))
+
```

Comment:
it might be clearer here to say "error loading %s: ..." % file, rather than talking about the handler object. Or to be sure the text of handler.error already includes the file and `die("%s\n%s" % (handler.error, handler.error_detail))` maybe

anyway up to you how much to refine now vs later, clearly this is better than nothing